apps/src/templates/sectionProgress/SectionProgressToggle.jsx

```diff
@@ -56,9 +64,15 @@ class SectionProgressToggle extends React.Component {
       });
     } else {
       firehoseClient.putRecord({
-        study: 'teacher-dashboard',
-        study_group: 'react',
-        event: 'progress-summary'
+        study: ' teacher_dashboard_actions',
+        study_group: 'progress',
+        event: 'view_change_toggle',
+        user_id: this.props.userId,
+        data_json: JSON.stringify({
+          section_id: this.props.sectionId,
+          old_view: ViewType.DETAIL,
+          new_view: ViewType.SUMMARY
+        })
```

Comment:
Since this is a two-state toggle, can we condense these? Maybe something like one `logToggle` that gets called in both places and then only save new_view because we know what the old_view must've been through process of elimination?

Reply:
I think Amanda wanted it this way because this is the toggle we will be adding more things to in the future (Standards, etc) so this allows us to continue to use the same data format when that happens.